src/Microsoft.Framework.CommandLineUtils.Sources/CommandLine/AnsiConsole.cs

```diff
@@ -9,34 +9,19 @@ namespace Microsoft.Framework.Runtime.Common.CommandLine
 {
     internal class AnsiConsole
     {
-        private AnsiConsole(TextWriter writer, bool useConsoleColor)
+        private AnsiConsole(TextWriter writer)
         {
             Writer = writer;
-
-            _useConsoleColor = useConsoleColor;
-            if (_useConsoleColor)
-            {
-                OriginalForegroundColor = Console.ForegroundColor;
-            }
         }
 
         private int _boldRecursion;
-        private bool _useConsoleColor;
 
-        public static AnsiConsole GetOutput(bool useConsoleColor)
-        {
-            return new AnsiConsole(Console.Out, useConsoleColor);
-        }
+        public static AnsiConsole Output = new AnsiConsole(Console.Out);
```

Comment:
The original code: https://github.com/aspnet/dnx/commit/d751821e0ae182e826f33dea3e7520a5f34a0383#diff-70f0f8d78afc1288add29e793af6cf3dR18
We could change it like you said though.